apps/innovations/_types/innovation.types.ts

```diff
@@ -21,12 +21,12 @@ export interface InnovationSectionModel {
   submittedAt: Date | null;
 }
 
-export type ReassessmentType =
-  | {
-      updatedInnovationRecord: CurrentCatalogTypes.catalogYesNo;
-      description: string;
-    }
-  | { description: string };
+export type ReassessmentType = {
+  reassessmentReason: string[];
```

Comment:
Shouldn't this have a type? With the valid values for this question?

Reply:
Makes sense, will do.